jormungandr/src/rest/v0/logic.rs

```diff
@@ -105,9 +105,12 @@ pub async fn get_account_state(
 
 pub async fn get_message_logs(context: &Context) -> Result<Vec<FragmentLog>, Error> {
     let logger = context.logger()?.new(o!("request" => "message_logs"));
-    let (reply_handle, reply_future) = intercom::unary_reply(logger);
+    let (reply_handle, reply_future) = intercom::unary_reply(logger.clone());
     let mut mbox = context.try_full()?.transaction_task.clone();
-    mbox.send(TransactionMsg::GetLogs(reply_handle)).await;
+    mbox.send(TransactionMsg::GetLogs(reply_handle))
+        .await
+        .map_err(|e| debug!(&logger, "error getting message logs"; "reason" => %e))
+        .ok();
     reply_future.await.map_err(Into::into)
```

Comment:
If we failed to send the message to the task, the reply will never get received.
Better return the error after logging if `mbox.send` fails; I think it's OK to convert it to status 500 for the response.

Reply:
@mzabaluev , how to wrap or transform the Error here?

Reply:
There is `TxMsgSendError` already, it just does not seem to be used.

Reply:
It is being used actually. In the calls to `try_send`. Which differs in returning type with `send`.